core/shared/i18n/translations/en.json

```diff
@@ -345,7 +345,8 @@
             "posts": {
                 "postNotFound": "Post not found.",
                 "invalidEmailRecipientFilter": "Invalid filter in email_recipient_filter param.",
-                "invalidVisibilityFilter": "Invalid filter in visibility property"
+                "invalidVisibilityFilter": "Invalid filter in visibility property",
```

Comment:
Worth burning these inf favor of "messages" object pattern that was [introduced recently](https://github.com/TryGhost/Utils/commit/e93164402536d8d43010bc4f62d2748254406955) :wink: